utils/src/ooinstall/variants.py

```diff
@@ -49,9 +49,16 @@ def latest_version(self):
     ]
 )
 
-# Ordered list of variants we can install, first is the default.
-SUPPORTED_VARIANTS = (OSE, AEP)
+origin = Variant('origin', 'OpenShift Origin',
+    [
+        Version('1.2', 'origin'),
+        Version('1.1', 'origin')
```

Comment:
CentOS released origin 1.1.6, which was basically a 1.2.0 preview.  We released it in CentOS because we were setting things up and needed to release something. 
I'm fine either way.  As I write up my CentOS Documentation I'm pretending 1.2 is the first.